src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -38,20 +39,30 @@ export class DownloadCLIClientComponent implements OnInit {
       (resultFromApi: Metadata) => {
         apiVersion = resultFromApi.version;
         this.dockstoreVersion = `${apiVersion}`;
-        this.downloadCli = `https://github.com/dockstore/dockstore/releases/download/${apiVersion}/dockstore`;
-        this.metadataService
-          .getRunnerDependencies(apiVersion, '3', 'cwltool', 'json')
-          .pipe(finalize(() => this.generateMarkdown()))
-          .subscribe(
-            (json: any) => {
-              if (json) {
-                this.cwltoolVersion = json.cwltool;
-              }
-            },
-            (err) => {
-              console.log('Unable to retrieve requirements.txt file.');
+        this.metadataService.getCliVersion().subscribe(
+          (json: any) => {
```

Comment:
Avoid `any`; it's a string not JSON (but probably should be JSON per Steve's comment in your web service PR).

Reply:
Fixed now uses CLIInfo class